websites/C/Character.AI/presence.ts

```diff
@@ -0,0 +1,94 @@
+const presence = new Presence({
+		clientId: "939893082546114611",
+	}),
+	logo = "https://i.imgur.com/fvjZLpL.png",
+	button = {
+		label: "View Page",
+		url: document.location.href,
+	},
```

Comment:
think its better to define this button const in the updatedata event, otherwise it wouldn't get updated on url replacements.